app/models/private_message.rb

```diff
@@ -0,0 +1,6 @@
+class PrivateMessage < ActiveRecord::Base
+  belongs_to :notifiable, polymorphic: true
+  belongs_to :user
+  belongs_to :ticket
+  default_scope {order('created_at DESC')}
```

Comment:
Space missing inside {.<br>Space missing inside }.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.